celavi/pylca_celavi/pylca_opt_background.py

```diff
@@ -386,7 +386,7 @@ def pyomo_postprocess(options=None, instance=None, results=None):
         # pyomo command-line.  For example:  python transport.py
     
         # This emulates what the pyomo command-line tools does
-        opt = SolverFactory("glpk")
+        opt = SolverFactory("ipopt")
```

Comment:
Do any changes need to be made in import statements and/or the setup file to use ipopt instead of glpk?

(Also I just want to double check - we definitely want to switch solvers?)

Reply:
I changed it because there was an issue with the optimization. However, it was resolved due to a separate solution rather than changing the solver. I think we can change it back. Let me check.

Reply:
Optimization failure issue resolved.